src/main/java/com/revature/controller/ProfileController.java

```diff
@@ -63,8 +64,13 @@ public boolean insertProfileInfo(@RequestBody Profile profile) {
 	 the page to search for profiles.
      * @param page An integer identifier the page to search for profiles.
      * @return A list of all candidate profiles on a given page. */
-	public List<Profile> getAllPaged(@PathVariable int page) {
-		return profileService.getAllProfilesPaged(page);
+	public List<Profile> getAllPaged(@PathVariable int page, @RequestHeader("usefilter") boolean useFilter,

```

Comment:
Use QueryParams.

Reply:
we will use QueryParams INSTEAD OF RequestHeader

Reply:
fixed